src/main/java/net/schmizz/sshj/common/KeyType.java

```diff
@@ -174,15 +174,15 @@ protected boolean isMyType(Key key) {
     },
 
     ED25519("ssh-ed25519") {
-        private final Logger logger = LoggerFactory.getLogger(KeyType.class);
+        private final Logger log = LoggerFactory.getLogger(KeyType.class);
```

Comment:
There are a few places — like these static initializers — where it’s not possible to inject a LoggerFactory.  I could use the new LoggerFactory.DEFAULT instead, but instead I just left them as-is, as the result is identical.

> On Aug 24, 2016, at 6:39 AM, Jeroen van Erp <notifications@github.com> wrote:
> 
> In src/main/java/net/schmizz/sshj/common/KeyType.java <https://github.com/hierynomus/sshj/pull/267#discussion_r76040618>:
> 
> > @@ -174,15 +174,15 @@ protected boolean isMyType(Key key) {
> >      },
> >  
> >      ED25519("ssh-ed25519") {
> > -        private final Logger logger = LoggerFactory.getLogger(KeyType.class);
> > +        private final Logger log = LoggerFactory.getLogger(KeyType.class);
> Isn't this using the wrong LoggerFactory now?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/hierynomus/sshj/pull/267/files/f63a88ec9f1ca21db2d18eb68056fcf51a107ff5#r76040618>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABIPfWE18KqAoYxuTzYWKzAfdlJvq0IWks5qjC1WgaJpZM4JrhJ->.
>